google_project/main.tf

```
@@ -29,4 +29,34 @@ resource "google_project_service" "project" {
   project            = local.project_id
   service            = each.key
   disable_on_destroy = false
-}
\ No newline at end of file
+}
+
+resource "google_project_iam_audit_config" "data_access_high" {
+  count = var.risk_level == "high" ? 1 : 0
+
+  project = local.project_id
+  service = "allServices"
+  audit_log_config {
+    log_type = "ADMIN_READ"
+  }
+  audit_log_config {
+    log_type = "DATA_READ"
+  }
+  audit_log_config {
+    log_type = "DATA_WRITE"
+  }
+}
+
+resource "google_logging_project_exclusion" "data_access_exclusions" {
+
+  name        = "exclude-data-access-log-sink"
+  description = "Exclude data access logs except BigQuery, secrets manager, and IAM for this project"
+
+  filter = <<EOT
+cloudaudit.googleapis.com/data_access
```

```suggestion
log_id("cloudaudit.googleapis.com/data_access")
```
Should we wrap this with log_id?